ansible/roles/containerd/tasks/install.yaml

```diff
@@ -43,3 +43,4 @@
     force: yes
   loop:
     - ctr
+    - containerd
```

Comment:
LGTM.  minor suggestion: The task name and comment description is specific to `ctr`. can add info about `containerd`